hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconAndAdminContainerCLI.java

```diff
@@ -330,34 +363,21 @@ private static void compareRMReportToReconResponse(String containerState)
     long rmMisReplCounter = rmReport.getStat(
         ReplicationManagerReport.HealthState.MIS_REPLICATED);
 
-    // Both threads are running every 1 second.
-    // Wait until all values are equal.
-    GenericTestUtils.waitFor(
-            () -> rmMissingCounter == reconResponse.getMissingCount() &&
```

Comment:
I see, without getting a new instance of the stats inside the waitFor, the value is always fixed. Thanks for catching that.